File: MORM.CSharp/Models/Venda/Transimposto.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("TRANSIMPOSTO")]
    public class Transimposto : CollectionItem
    {
        [Campo("ID_TRANSACAO", CampoTipo.Key)]
        public string Id_Transacao { get; set; }
        [Campo("NR_ITEM", CampoTipo.Key)]
        public int Nr_Item { get; set; }
        [Campo("CD_IMPOSTO", CampoTipo.Key)]
        public int Cd_Imposto { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("PR_ALIQUOTA", CampoTipo.Req)]
        public double Pr_Aliquota { get; set; }
        [Campo("VL_BASECALCULO", CampoTipo.Req)]
        public double Vl_Basecalculo { get; set; }
        [Campo("PR_BASECALCULO", CampoTipo.Req)]
        public double Pr_Basecalculo { get; set; }
        [Campo("PR_REDBASECALCULO", CampoTipo.Req)]
        public double Pr_Redbasecalculo { get; set; }
        [Campo("VL_IMPOSTO", CampoTipo.Req)]
        public double Vl_Imposto { get; set; }
        [Campo("VL_OUTRO", CampoTipo.Req)]
        public double Vl_Outro { get; set; }
        [Campo("VL_ISENTO", CampoTipo.Req)]
        public double Vl_Isento { get; set; }
        [Campo("CD_CST", CampoTipo.Req)]
        public string Cd_Cst { get; set; }
        [Campo("CD_CSOSN", CampoTipo.Nul)]
        public string Cd_Csosn { get; set; }
    }
}

File: MORM.Java/src/MORM/Java/Models/Venda/Transimposto.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "TRANSIMPOSTO")
public class Transimposto extends CollectionItem
{
    @CampoAttribute(Campo = "ID_TRANSACAO", Tipo = CampoTipo.Key)
    public String Id_Transacao;
    @CampoAttribute(Campo = "NR_ITEM", Tipo = CampoTipo.Key)
    public Integer Nr_Item;
    @CampoAttribute(Campo = "CD_IMPOSTO", Tipo = CampoTipo.Key)
    public Integer Cd_Imposto;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "PR_ALIQUOTA", Tipo = CampoTipo.Req)
    public Float Pr_Aliquota;
    @CampoAttribute(Campo = "VL_BASECALCULO", Tipo = CampoTipo.Req)
    public Float Vl_Basecalculo;
    @CampoAttribute(Campo = "PR_BASECALCULO", Tipo = CampoTipo.Req)
    public Float Pr_Basecalculo;
    @CampoAttribute(Campo = "PR_REDBASECALCULO", Tipo = CampoTipo.Req)
    public Float Pr_Redbasecalculo;
    @CampoAttribute(Campo = "VL_IMPOSTO", Tipo = CampoTipo.Req)
    public Float Vl_Imposto;
    @CampoAttribute(Campo = "VL_OUTRO", Tipo = CampoTipo.Req)
    public Float Vl_Outro;
    @CampoAttribute(Campo = "VL_ISENTO", Tipo = CampoTipo.Req)
    public Float Vl_Isento;
    @CampoAttribute(Campo = "CD_CST", Tipo = CampoTipo.Req)
    public String Cd_Cst;
    @CampoAttribute(Campo = "CD_CSOSN", Tipo = CampoTipo.Nul)
    public String Cd_Csosn;
}